src/translate/highcharts/scanner.js

```diff
@@ -1,4 +1,13 @@
-export default function highcharts(results, options) {
+function informAboutDependencies(){
```

Comment:
What do you think about putting each of these in their own directory, with documentation and image?

Reply:
documentation is definitely necessary, I'll throw together another PR for this pretty soon